### ISSUE TITLE
Consider overview accessibility help dialog

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#223786]
We have accessibility help dialogs per feature area, actionable when that feature is focused.

I am wondering if we should have a help dialog which contains all of this info in one place. 

We could add a go to symbol command for navigation.

Would that be useful @rperez030 and @jooyoungseo?

cc @isidorn

**Comment by meganrogge** [comment]
We now have the accessibility getting started walkthrough, which serves this purpose.
